src/common/proxy_wasm/ngx_proxy_wasm_properties.c

```diff
@@ -171,6 +171,23 @@ get_request_time(ngx_proxy_wasm_ctx_t *pwctx, ngx_str_t *path,
 }
 
 
+static ngx_int_t
+is_subrequest(ngx_proxy_wasm_ctx_t *pwctx, ngx_str_t *path,
+    ngx_str_t *value)
+{
+    ngx_http_wasm_req_ctx_t  *rctx = (ngx_http_wasm_req_ctx_t *) pwctx->data;
+    ngx_http_request_t       *r = rctx->r;
+
+    if (r == r->main) {
+        ngx_str_set(value, "false");
+    } else {
```

Comment:
The linter will also complain about this `else` which is missing the blank line above. We should actually do this logic like `connection.mtls`, and move the `static` `true/false` strings outside of the mtls getter so that we can have the same `value = r == r->main ? true : false` logic (see `connection.mtls` getter).